### ISSUE TITLE
TensorFlow Model Does Not Restore Reliably

### ISSUE THREAD

**Issue by dsilver829** [udacity/sdc-issue-reports#576]
https://github.com/ahnguyen09/Car-ND-Projects/tree/master/CarND-Traffic-Sign-Classifier-Project

1) load data at begining, including the few cells below that includes numpy and mat plot lib (edited)

2) Skip to heading with pre-process data and run the import and processing functions

3) Skip to right before model architecture to load X_train_mod from pickle

4) Load TF under model architecture. Then run the model layer function (def stemBlock...)

5) Run the Placeholder and training pipeline code with logits, cross_entropy. Also run model evaluation because we will use the evaluate function

6) Skip to test model on new image and run first script to load in internet images

**Comment by dsilver829** [comment]
An Nguyen on Slack has additional information on this.

**Comment by baumanab** [comment]
Closing as previously addressed, per recent conversations.
